release-notes/6.0/releases.json

```diff
@@ -4,6 +4,7 @@
   "latest-release-date": "2022-10-11",
   "latest-runtime": "6.0.10",
   "latest-sdk": "6.0.402",
+  "release-type" : "lts",
   "support-phase": "lts",
```

Comment:
@mairaw - release-type should be 'active'. Is it OK to make that change for the 'catch-up' merge today?

Reply:
yes, I've been testing to what's in the private repo so those are the changes I'm expecting to see and I'm prepared to handle